### ISSUE TITLE
Max execution time of 60 seconds exceeded  in CLI

### ISSUE THREAD

**Issue by kassner** [magento/magento2#2461]
Hi,

I got this error when running `php bin/magento setup:static-content:deploy` from the CLI. I have to run with `-d max_executin_time=0` to avoid the error. This should not happen since PHP CLI has `max_execution_time` with `0` as default.

IMHO, we shouldn't define the `max_execution_time` or call `set_time_limit` within the code, but let the php.ini values kick in or at least define it in `static.php`.

```
root@bad6c70cf905:/var/www# sudo -H -u www-data /usr/bin/php bin/magento setup:static-content:deploy
Requested languages: en_US
=== frontend -> Magento/luma -> en_US ===
........................PHP Fatal error:  Maximum execution time of 60 seconds exceeded in /var/www/vendor/oyejorge/less.php/lib/Less/Parser.php on line 1539
PHP Stack trace:
PHP   1. {main}() /var/www/bin/magento:0
PHP   2. Symfony\Component\Console\Application->run() /var/www/bin/magento:25
PHP   3. Magento\Framework\Console\Cli->doRun() /var/www/vendor/symfony/console/Symfony/Component/Console/Application.php:126
PHP   4. Symfony\Component\Console\Application->doRun() /var/www/lib/internal/Magento/Framework/Console/Cli.php:49
PHP   5. Symfony\Component\Console\Application->doRunCommand() /var/www/vendor/symfony/console/Symfony/Component/Console/Application.php:195
PHP   6. Symfony\Component\Console\Command\Command->run() /var/www/vendor/symfony/console/Symfony/Component/Console/Application.php:874
PHP   7. Magento\Deploy\Console\Command\DeployStaticContentCommand->execute() /var/www/vendor/symfony/console/Symfony/Component/Console/Command/Command.php:257
PHP   8. Magento\Deploy\Model\Deployer->deploy() /var/www/app/code/Magento/Deploy/Console/Command/DeployStaticContentCommand.php:120
PHP   9. Magento\Deploy\Model\Deployer->deployFile() /var/www/app/code/Magento/Deploy/Model/Deployer.php:181
PHP  10. Magento\Framework\App\View\Asset\Publisher->publish() /var/www/app/code/Magento/Deploy/Model/Deployer.php:340
PHP  11. Magento\Framework\App\View\Asset\Publisher->publishAsset() /var/www/lib/internal/Magento/Framework/App/View/Asset/Publisher.php:50
PHP  12. Magento\Framework\View\Asset\File->getSourceFile() /var/www/lib/internal/Magento/Framework/App/View/Asset/Publisher.php:63
PHP  13. Magento\Framework\View\Asset\Source->getFile() /var/www/lib/internal/Magento/Framework/View/Asset/File.php:150
PHP  14. Magento\Framework\View\Asset\Source->preProcess() /var/www/lib/internal/Magento/Framework/View/Asset/Source.php:86
PHP  15. Magento\Framework\View\Asset\PreProcessor\Pool->process() /var/www/lib/internal/Magento/Framework/View/Asset/Source.php:128
PHP  16. Magento\Developer\Model\View\Asset\PreProcessor\PreprocessorStrategy->process() /var/www/lib/internal/Magento/Framework/View/Asset/PreProcessor/Pool.php:74
PHP  17. Magento\Framework\View\Asset\PreProcessor\AlternativeSource->process() /var/www/app/code/Magento/Developer/Model/View/Asset/PreProcessor/PreprocessorStrategy.php:62
PHP  18. Magento\Framework\View\Asset\PreProcessor\AlternativeSource->processContent() /var/www/lib/internal/Magento/Framework/View/Asset/PreProcessor/AlternativeSource.php:115
PHP  19. Magento\Framework\Css\PreProcessor\Adapter\Less\Processor->processContent() /var/www/lib/internal/Magento/Framework/View/Asset/PreProcessor/AlternativeSource.php:155
PHP  20. Less_Parser->getCss() /var/www/lib/internal/Magento/Framework/Css/PreProcessor/Adapter/Less/Processor.php:86
PHP  21. Less_Tree_Ruleset->compile() /var/www/vendor/oyejorge/less.php/lib/Less/Parser.php:187
PHP  22. Less_Tree_Ruleset->PrepareRuleset() /var/www/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:70
PHP  23. Less_Tree_Ruleset->evalImports() /var/www/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:235
PHP  24. Less_Tree_Import->compile() /var/www/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:248
PHP  25. Less_Tree_Import->ParseImport() /var/www/vendor/oyejorge/less.php/lib/Less/Tree/Import.php:193
PHP  26. Less_Tree_Ruleset->evalImports() /var/www/vendor/oyejorge/less.php/lib/Less/Tree/Import.php:273
PHP  27. Less_Tree_Import->compile() /var/www/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:248
PHP  28. Less_Tree_Import->ParseImport() /var/www/vendor/oyejorge/less.php/lib/Less/Tree/Import.php:193
PHP  29. Less_Parser->parseFile() /var/www/vendor/oyejorge/less.php/lib/Less/Tree/Import.php:269
PHP  30. Less_Parser->GetRules() /var/www/vendor/oyejorge/less.php/lib/Less/Parser.php:343
PHP  31. Less_Parser->parsePrimary() /var/www/vendor/oyejorge/less.php/lib/Less/Parser.php:532
PHP  32. Less_Parser->MatchFuncs() /var/www/vendor/oyejorge/less.php/lib/Less/Parser.php:860
PHP  33. Less_Parser->parseMixinDefinition() /var/www/vendor/oyejorge/less.php/lib/Less/Parser.php:710
```

Thanks!

**Comment by mazhalai** [comment]
@kassner can you please be more specific? `max_execution_time` has not been set in code, so what exactly is the issue.

**Comment by kassner** [comment]
@mazhalai To be fair, the `max_execution_time` is not set in Magento code itself, but in the [cssmin](https://github.com/tubalmartin/YUI-CSS-compressor-PHP-port/blob/v2.4.8-p4/cssmin.php#L191-L208) package, and it effects the entire execution of `bin/magento setup:static-content:deploy`. Since Magento is using this package [intentionally](https://github.com/magento/magento2/blob/2.0.0/composer.json#L45), I think Magento should mitigate the effects.

**Comment by mazhalai** [comment]
@kassner thank you for clarifying.

**Comment by shiftedreality** [comment]
@kassner 

Thank you for pointing this issue. Internal ticket MAGETWO-47892 was created for investigation

**Comment by keithbentrup** [comment]
We've experienced this issue, too. For those who need to work past it for now until the resolution is merged, we've manually set `$this->max_execution_time = 600;` in  vendor/tubalmartin/cssmin/cssmin.php (overwriting the hardcoded value of 60).

**Comment by dsikkema-magento** [comment]
@keithbentrup, @kassner I'm having some trouble reproducing this. On line app/etc/di.xml:698, we disable CSSMin's option to modify settings, so for me, changing the value there does nothing for me.

Can you check what is the value of $raise_php_limits in \CSSmin::__construct? And can you please check the value of ini_get('max_execution_time') in the bin/magento file itself?

**Comment by dsikkema-magento** [comment]
Closing this for now, if anyone knows what I'm missing to reproduce then please re-open.

**Comment by dsikkema-magento** [comment]
Just to be sure... _ping_.
This is a problem we definitely want to solve if someone is still able to reproduce it, so I just thought I'd give it one more shout here.

**Comment by whizkid79** [comment]
pong @dsikkema .

I expierence the same issue like @kassner when I enabled CSS minification in the backend. I did some debugging and found out that `\CSSmin::__construct` get handed a false normally. But if you run `bin/magento setup:di:compile` before `bin/magento setup:static-content:deploy` then `\CSSmin::__construct` is initialized with an instance of `Magento\Framework\ObjectManager\ObjectManager` which is casted to `bool` in the constructor which results in true.

It seems to be related to the compiled DI information.
I'm a bit lost in debugging this further. Any ideas would be appreciated.

**Comment by katmoon** [comment]
If anybody experiences the issue, try to rerun `bin/magento setup:static-content:deploy` command several times.

**Comment by dsikkema-magento** [comment]
@whizkid79 thank you for your investigation, when I run the compiler first I do see the same result.

**Comment by atIOCrON** [comment]
@dsikkema I also get the same issue. By following @katmoon advise to run the deployment several times, I can get it to finish. However it should still be addressed so it doesn't time out in the first place.

**Comment by gabrielqs** [comment]
I'm seeing this on 2.0.7. "-d maximum_execution_time=0" did the trick, but that really shouldn't be necessary. Only started seeing this after enabling minification.

**Comment by gabrielqs** [comment]
I'm sorry, it the -d flag didn't work at all... I had to disable the minification in order for the command to run, or to leave it on and deploy static content several times until it was able to finish it.

**Comment by Rui-Carvalho** [comment]
I did a grep with `grep -Rn "max_execution_time" .` on the project folder and found - as @whizkid79 mentioned - the culprit to be:
 `./vendor/tubalmartin/cssmin/cssmin.php:47:   $this->max_execution_time = 60; // 1 min`

So, edit this file temporarily and get the value to 600 - or whatever makes it work for you - at least until someone on their devteam handles this bit weird configuration in their next release.

**Comment by joshuafredrickson** [comment]
@katmoon's suggestion worked for me. I re-ran deployment 6 times before it successfully completed. Next time I'll give @Rui-Carvalho's solution a go.

**Comment by muhupower** [comment]
Hello,

When I deploy, I am getting this same error


```

Fatal error: Maximum function nesting level of '100' reached, aborting! in /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php on line 311

Call Stack:
    0.0003     238328   1. {main}() /home/ceilingcom/public_html/bin/magento:0
    1.2851   44906720   2. Symfony\Component\Console\Application->run() /home/ceilingcom/public_html/bin/magento:23
    1.2860   44952152   3. Magento\Framework\Console\Cli->doRun() /home/ceilingcom/public_html/vendor/symfony/console/Symfony/Component/Console/Application.php:126
    1.2860   44952408   4. Symfony\Component\Console\Application->doRun() /home/ceilingcom/public_html/vendor/magento/framework/Console/Cli.php:96
    1.2861   44953296   5. Symfony\Component\Console\Application->doRunCommand() /home/ceilingcom/public_html/vendor/symfony/console/Symfony/Component/Console/Application.php:195
    1.2861   44953768   6. Symfony\Component\Console\Command\Command->run() /home/ceilingcom/public_html/vendor/symfony/console/Symfony/Component/Console/Application.php:874
    1.2866   44958848   7. Magento\Deploy\Console\Command\DeployStaticContentCommand->execute() /home/ceilingcom/public_html/vendor/symfony/console/Symfony/Component/Console/Command/Command.php:257
    1.6095   45617384   8. Magento\Deploy\Console\Command\DeployStaticContentCommand->deploy() /home/ceilingcom/public_html/vendor/magento/module-deploy/Console/Command/DeployStaticContentCommand.php:373
    1.6095   45618768   9. Magento\Deploy\Model\Deployer->deploy() /home/ceilingcom/public_html/vendor/magento/module-deploy/Console/Command/DeployStaticContentCommand.php:448
    7.7648   57852312  10. Magento\Deploy\Model\Deployer->deployFile() /home/ceilingcom/public_html/vendor/magento/module-deploy/Model/Deployer.php:283
    7.7650   57853040  11. Magento\Framework\App\View\Asset\Publisher->publish() /home/ceilingcom/public_html/vendor/magento/module-deploy/Model/Deployer.php:445
    7.7651   57853232  12. Magento\Framework\App\View\Asset\Publisher->publishAsset() /home/ceilingcom/public_html/vendor/magento/framework/App/View/Asset/Publisher.php:61
    7.7651   57853232  13. Magento\Framework\View\Asset\File->getSourceFile() /home/ceilingcom/public_html/vendor/magento/framework/App/View/Asset/Publisher.php:73
    7.7651   57853232  14. Magento\Framework\View\Asset\Source->getFile() /home/ceilingcom/public_html/vendor/magento/framework/View/Asset/File.php:150
    7.7651   57853232  15. Magento\Framework\View\Asset\Source->preProcess() /home/ceilingcom/public_html/vendor/magento/framework/View/Asset/Source.php:105
    7.7655   57855048  16. Magento\Framework\View\Asset\PreProcessor\Pool->process() /home/ceilingcom/public_html/vendor/magento/framework/View/Asset/Source.php:152
    7.7709   58207200  17. Magento\Developer\Model\View\Asset\PreProcessor\PreprocessorStrategy->process() /home/ceilingcom/public_html/vendor/magento/framework/View/Asset/PreProcessor/Pool.php:74
    7.7712   58212656  18. Magento\Framework\View\Asset\PreProcessor\AlternativeSource->process() /home/ceilingcom/public_html/vendor/magento/module-developer/Model/View/Asset/PreProcessor/PreprocessorStrategy.php:62
    7.7713   58213512  19. Magento\Framework\View\Asset\PreProcessor\AlternativeSource->processContent() /home/ceilingcom/public_html/vendor/magento/framework/View/Asset/PreProcessor/AlternativeSource.php:115
    7.7730   58261040  20. Magento\Framework\Css\PreProcessor\Adapter\Less\Processor->processContent() /home/ceilingcom/public_html/vendor/magento/framework/View/Asset/PreProcessor/AlternativeSource.php:155
    8.1092   59673008  21. Less_Parser->getCss() /home/ceilingcom/public_html/vendor/magento/framework/Css/PreProcessor/Adapter/Less/Processor.php:88
    8.1093   59674736  22. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Parser.php:198
   16.6166  108295064  23. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:94
   16.6350  108357856  24. Less_Tree_Media->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:94
   16.6351  108361024  25. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Media.php:67
   16.6351  108362952  26. Less_Tree_Ruleset->EvalMixinCalls() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:88
   16.6351  108363008  27. Less_Tree_Mixin_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:147
   16.6456  108369704  28. Less_Tree_Mixin_Definition->evalCall() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Call.php:127
   16.6457  108378160  29. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Definition.php:175
   16.6457  108379008  30. Less_Tree_Ruleset->EvalMixinCalls() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:88
   16.6457  108379056  31. Less_Tree_Mixin_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:147
   16.6487  108383792  32. Less_Tree_Mixin_Definition->evalCall() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Call.php:127
   16.6487  108390264  33. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Definition.php:175
   16.6488  108391136  34. Less_Tree_Ruleset->EvalMixinCalls() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:88
   16.7990  108629056  35. Less_Tree_Mixin_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:147
   16.7993  108633952  36. Less_Tree_Mixin_Definition->evalCall() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Call.php:127
   16.7994  108644280  37. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Definition.php:175
   16.7995  108645232  38. Less_Tree_Ruleset->EvalMixinCalls() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:88
   16.8015  108652192  39. Less_Tree_Mixin_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:147
   16.8018  108657912  40. Less_Tree_Mixin_Definition->evalCall() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Call.php:127
   16.8019  108669272  41. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Definition.php:175
   16.8019  108670080  42. Less_Tree_Ruleset->EvalMixinCalls() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:88
   16.8039  108677000  43. Less_Tree_Mixin_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:147
   16.8042  108682504  44. Less_Tree_Mixin_Definition->evalCall() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Call.php:127
   16.8043  108694296  45. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Definition.php:175
   16.8043  108695200  46. Less_Tree_Ruleset->EvalMixinCalls() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:88
   16.8063  108701952  47. Less_Tree_Mixin_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:147
   16.8066  108707320  48. Less_Tree_Mixin_Definition->evalCall() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Call.php:127
   16.8067  108850968  49. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Definition.php:175
   16.8067  108851816  50. Less_Tree_Ruleset->EvalMixinCalls() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:88
   16.8087  108858656  51. Less_Tree_Mixin_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:147
   16.8090  108863920  52. Less_Tree_Mixin_Definition->evalCall() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Call.php:127
   16.8091  108877120  53. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Definition.php:175
   16.8091  108877920  54. Less_Tree_Ruleset->EvalMixinCalls() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:88
   16.8112  108884848  55. Less_Tree_Mixin_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:147
   16.8115  108890272  56. Less_Tree_Mixin_Definition->evalCall() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Call.php:127
   16.8116  108904688  57. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Definition.php:175
   16.8116  108905536  58. Less_Tree_Ruleset->EvalMixinCalls() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:88
   16.8137  108912512  59. Less_Tree_Mixin_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:147
   16.8140  108917936  60. Less_Tree_Mixin_Definition->evalCall() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Call.php:127
   16.8141  108933224  61. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Definition.php:175
   16.8141  108934152  62. Less_Tree_Ruleset->EvalMixinCalls() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:88
   16.8163  108941024  63. Less_Tree_Mixin_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:147
   16.8167  108946472  64. Less_Tree_Mixin_Definition->evalCall() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Call.php:127
   16.8168  108962552  65. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Definition.php:175
   16.8169  108963392  66. Less_Tree_Ruleset->EvalMixinCalls() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:88
   16.8190  108970208  67. Less_Tree_Mixin_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:147
   16.8193  108975568  68. Less_Tree_Mixin_Definition->evalCall() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Call.php:127
   16.8194  108992064  69. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Definition.php:175
   16.8194  108992880  70. Less_Tree_Ruleset->EvalMixinCalls() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:88
   16.8215  108999752  71. Less_Tree_Mixin_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:147
   16.8219  109005040  72. Less_Tree_Mixin_Definition->evalCall() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Call.php:127
   16.8219  109022288  73. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Definition.php:175
   16.8220  109023088  74. Less_Tree_Ruleset->EvalMixinCalls() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:88
   16.8240  109029920  75. Less_Tree_Mixin_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:147
   16.8244  109035280  76. Less_Tree_Mixin_Definition->evalCall() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Call.php:127
   16.8245  109053232  77. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Definition.php:175
   16.8245  109054072  78. Less_Tree_Ruleset->EvalMixinCalls() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:88
   16.8266  109060896  79. Less_Tree_Mixin_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:147
   16.8270  109066216  80. Less_Tree_Mixin_Definition->evalCall() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Call.php:127
   16.8271  109084888  81. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Definition.php:175
   16.8271  109085688  82. Less_Tree_Ruleset->EvalMixinCalls() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:88
   16.8293  109092568  83. Less_Tree_Mixin_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:147
   16.8297  109097944  84. Less_Tree_Mixin_Definition->evalCall() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Call.php:127
   16.8298  109117312  85. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Definition.php:175
   16.8298  109118128  86. Less_Tree_Ruleset->EvalMixinCalls() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:88
   16.8298  109249160  87. Less_Tree_Mixin_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:147
   16.8315  109254944  88. Less_Tree_Mixin_Definition->evalCall() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Call.php:127
   16.8316  109276008  89. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Mixin/Definition.php:175
   16.8316  109276808  90. Less_Tree_Ruleset->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:94
   16.8317  109280544  91. Less_Tree_Rule->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php:94
   16.8318  109280544  92. Less_Tree_Value->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Rule.php:75
   16.8318  109280752  93. Less_Tree_Expression->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Value.php:27
   16.8318  109280920  94. Less_Tree_Call->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Expression.php:52
   16.8318  109281056  95. Less_Tree_Expression->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Call.php:45
   16.8318  109281200  96. Less_Tree_Expression->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Expression.php:52
   16.8318  109281360  97. Less_Tree_Operation->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Expression.php:52
   16.8318  109281360  98. Less_Tree_Variable->compile() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Operation.php:30
   16.8318  109281352  99. Less_Tree_Ruleset->variable() /home/ceilingcom/public_html/vendor/oyejorge/less.php/lib/Less/Tree/Variable.php:42


```

Please help..

I changed it to 600, still getting same results.

I have reran the deploy over and over again, and same results.

The backend has no css or js, so I cant get into their and disable it, is there a command where I can disable the minifiy?

Thanks,

**Comment by kassner** [comment]
@muhupower this is not related to this issue, but with the fact that you have xdebug enabled. Please refer to http://devdocs.magento.com/guides/v2.1/install-gde/trouble/tshoot_install-issues.html fix it.

**Comment by ngoctruong1610** [comment]
- I met similar problem with my projects, any update on this ?

**Comment by AntonEvers** [comment]
I've created this pull request to resolve the problem. There were two pull requests already, but neither of them fixed the problem completely.

https://github.com/tubalmartin/YUI-CSS-compressor-PHP-port/pull/36

**Comment by AntonEvers** [comment]
@tubalmartin has merged the pull request in version v2.4.8-p6. https://github.com/magento/magento2/pull/8959 requires this version and fixes the problem in Magento.

**Comment by tubalmartin** [comment]
BTW, since the version of cssmin is fixed in composer.json file, I suggest you update it to the latest version of Cssmin released today which is v2.4.8-p8. It's the most stable and reliable so far ;) Enjoy!

**Comment by tubalmartin** [comment]
Just to let you know guys, as the maintainer of YUI-CSS-compressor-PHP-port, that I've just released v3.0.0 which is the most stable release to date. Updating to this version is strongly recommended.

**Comment by AntonEvers** [comment]
Thanks for the heads-up!
